include/gatekeeper_fib.h

```diff
@@ -227,26 +227,54 @@ struct ip_prefix {
 	int           len;
 };
 
-struct gk_fib_dump_entry {
+/*
+ * Since GK_FWD_GRANTOR entries can have mulitple Grantor IPs
+ * for load balancing (and therefore multiple next hops),
+ * we group together this information into an address set.
+ */
+struct fib_addr_set {
```

Comment:
Shouldn't we add the tag `dump` to the name of this struct to emphasize that is meant to be used for dumps and not internally?